cli/internal/action/checkDiff.go

```diff
@@ -29,41 +32,36 @@ type CheckDiffArgs struct {
 	Output        string
 }
 
-type CheckDiffOutput struct {
-	Recommendations []CheckDiffRecommendation `json:"recommendations"`
+type CheckOutput struct {
+	Recommendations []CheckRecommendation `json:"recommendations"`
+	Head            string                `json:"head"`
+	Base            string                `json:"base"`
 }
 
-type CheckDiffRecommendation struct {
-	Source         string   `json:"documentationSource"`
-	Document       string   `json:"document"`
-	Section        []string `json:"section,omitempty"`
```

Comment:
Let's talk about this change IRL. I _think_ we want to omit empty here still?

Reply:
Per an IRL conversation, we'll remove the omitempty so that section is always present.